modules/fbs/includes/fbs.reservation.inc

```diff
@@ -357,9 +361,13 @@ function fbs_reservation_options($account, $default_branch = NULL) {
     $preferred_branch = $default_branch;
   }
 
+  $profile = ding_user_provider_profile($account);
```

Comment:
So close :-)

Why load the profile here, when the $profile is only used inside fbs_reservation_default_interest_period();

So why not: 

`$interest_period = fbs_reservation_default_interest_period($account);`

Reply:
I agree. Passing the profile to `fbs_reservation_default_interest_period` does not make any sense when the function expects an account as the argument and will load the profile based on the passed account (the same as this line does) on L395.